nucleus/payara-modules/nucleus-microprofile/config-service/src/main/java/fish/payara/nucleus/microprofile/config/spi/ConfigExpressionResolver.java

```diff
@@ -121,10 +130,17 @@ private ConfigValueImpl getValue(String propertyName) {
         for (ConfigSource source : sources) {
             final String result = source.getValue(propertyName);
             if (result != null && !result.isEmpty()) {
+                String resolvedExpression = null;
+                try {
+                    resolvedExpression = resolveExpression(result);
+                } catch(NoSuchElementException noSuchElementException) {
+                    log.warn("We don't need to thrown the exception just wrap the null result " +
```

Comment:
The message sounds strange, explaining what we don't need to do...?
I'd suggest message like "Using null value in configuration, expression %s".

Reply:
already done, thanks